modules/authorization-manager/src/main/java/org/opencastproject/authorization/xacml/manager/endpoint/AbstractAclServiceRestEndpoint.java

```diff
@@ -304,100 +288,6 @@ public Response deleteAcl(@PathParam("aclId") long aclId) throws NotFoundExcepti
-              type = INTEGER
-          )
-      },
-      responses = {
-          @RestResponse(responseCode = SC_OK, description = "The ACL has been successfully applied"),
-          @RestResponse(responseCode = SC_NOT_FOUND, description = "The ACL or the episode has not been found"),
-          @RestResponse(responseCode = SC_INTERNAL_SERVER_ERROR, description = "Internal error")
-      }
-  )
-  public Response applyAclToEpisode(@PathParam("episodeId") String episodeId, @FormParam("aclId") Long aclId) {
-    final AclService aclService = aclService();
-    final Option<Option<ManagedAcl>> macl = option(aclId).map(getManagedAcl(aclService));
-    if (macl.isSome() && macl.get().isNone()) {
-      return notFound();
-    }
-    try {
-      if (aclService.applyAclToEpisode(episodeId, Options.join(macl))) {
-        return ok();
-      } else {
-        return notFound();
-      }
-    } catch (AclServiceException e) {
-      logger.error("Error applying acl to episode {}", episodeId);
-      return serverError();
-    }
-  }
-
-  @POST
-  @Path("/apply/series/{seriesId}")
-  @RestQuery(
-      name = "applyAclToSeries",
-      description = "Immediate application of an ACL to a series",
-      returnDescription = "Status code",
-      pathParameters = {
-          @RestParameter(name = "seriesId", isRequired = true, description = "The series ID", type = STRING)
-      },
-      restParameters = {
-          @RestParameter(
-              name = "aclId",
-              isRequired = true,
-              description = "The ID of the ACL to apply",
-              type = INTEGER
-          ),
-          @RestParameter(
-              name = "override",
-              isRequired = false,
-              defaultValue = "false",
-              description = "If true the series ACL will take precedence over any existing episode ACL",
-              type = STRING
-          )
-      },
-      responses = {
-          @RestResponse(responseCode = SC_OK, description = "The ACL has been successfully applied"),
-          @RestResponse(responseCode = SC_NOT_FOUND, description = "The ACL or the series has not been found"),
-          @RestResponse(responseCode = SC_INTERNAL_SERVER_ERROR, description = "Internal error")
-      }
-  )
-  public Response applyAclToSeries(
-      @PathParam("seriesId") String seriesId,
-      @FormParam("aclId") long aclId,
-      @DefaultValue("false") @FormParam("override") boolean override
-  ) {
-    final AclService aclService = aclService();
-    for (ManagedAcl macl : aclService.getAcl(aclId)) {
-      try {
-        if (aclService.applyAclToSeries(seriesId, macl, override)) {
-          return ok();
-        } else {
-          return notFound();
-        }
-      } catch (AclServiceException e) {
-        logger.error("Error applying acl to series {}", seriesId);
-        return serverError();
-      }
-    }
-    // acl not found
-    return notFound();
-  }
-
```

Comment:
This should be discussed in the technical meeting, as this is changing the API. It is an internal API and 10.x is still not released, so I'm not sure this qualifies for a minor release patch.

Reply:
This was discussed in the Technical Meeting. The Series ACL endpoint has to be removed for this PR and has a good replacement among the series service endpoints, so this was approved. The Episode ACL endpoint is used by the stud.iP integration however. Since I mostly removed it for consistency, I have decided to re-add it to make their lives easier, though we should think about a replacement in another spot for the future.